### PR TITLE
Bug 1192834: disable compact backups in 2.4

### PR DESCRIPTION
**Pull request by gl-sergei**
Merge test cases for bug 1192834, but disable them. Change `compact.sh` to check that `--compact` option no longer available.

### CODE REVIEW

**Comment by gl-sergei** [comment]
http://jenkins.percona.com/job/percona-xtrabackup-2.4-param-medium/10/

File: storage/innobase/xtrabackup/test/t/bug1192834-purge.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+skip_test "Enable when bug #1192834 is fixed in 2.4"
```
Comment (VasilyNemkov) [inline storage/innobase/xtrabackup/test/t/bug1192834-purge.sh:7]: If I get it right, this PR would fix the 1192834. Should we enable this test now?

File: storage/innobase/xtrabackup/test/t/bug1192834-rollback.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+skip_test "Enable when bug #1192834 is fixed in 2.4"
```
Comment (VasilyNemkov) [inline storage/innobase/xtrabackup/test/t/bug1192834-rollback.sh:7]: Same here, enable test?

**Review by VasilyNemkov — COMMENTED** [review]
Mostly Ok, but please take a second look at the tests

**Comment by gl-sergei** [comment]
I was going to mark bug 1192834 as "Won't fix" in 2.4. But was to going to keep the tests in case if we change our mind. The bug will still be valid for 2.4 even though one will not be able to take compact backup with 2.4, he can take the backup using 2.3 and crash 2.4

**Pull request merged**